examples/tasks/run-mock-psutil.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# get the directory the script exists in
+__dir="$(cd ../../scripts/docker/large && pwd)"
```

Comment:
not able to run it from the project root directory. Should this be the absolute path from the root of the project instead of a relative path?